apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -3312,8 +3312,9 @@ export const extensions: Extensions = {
                         },
                         usernameHint: "Must be an alphanumeric (a-z, A-Z, 0-9) string between {{minLength}} to " +
                             "{{maxLength}} characters including at least one letter.",
-                        usernameSpecialCharHint: "Must be a unique string between {{minLength}} to " +
-                            "{{maxLength}} characters including at least one letter.",
+                        usernameSpecialCharHint: "Must be {{minLength}} to {{maxLength}} characters long and " +
```

Comment:
Shall we update the translations?

Reply:
Addressed https://github.com/wso2/identity-apps/pull/4819/commits/918e18032f77540c321fb9192a43e4fa5516421c